src/Tickets/Commerce/Hooks.php

```diff
@@ -416,4 +420,12 @@ public function add_display_post_states( $post_states, $post ) {
 		return $post_states;
 	}
 
+	public function skip_sanitization( $meta_value, $meta_key ) {
```

Comment:
Doc block. functionality should not live in the Hooks class, so create a method on the `Settings` and use a filter here to point to it. See the method `filter_registration_cart_provider`